client/my-sites/themes/index.web.js

```diff
@@ -6,7 +6,8 @@ import userFactory from 'lib/user';
 import { makeLayout } from 'controller';
 import { makeNavigation, siteSelection } from 'my-sites/controller';
 import { singleSite, multiSite, loggedOut } from './controller';
-import { getSubjects } from './theme-filters.js';
+import { getSubjects } from './theme-filters';
+import validateFilters from './validate-filters';
```

Comment:
Trouble is, `./controller.jsx` is isomorphic, whereas this new middleware needs `page.js`.

Really, we want this new middleware to be isomorphic, but we don't have an isomorphic way to handle re-directs yet.